### PR TITLE
docs(global): Optimize and improve the readme.zh-CN.md.

### PR DESCRIPTION
**Pull request by chenying2929**
1. Optimize and restructure the structure (9 parts): intro, composition, features, documentations, usage methods, contribution, collaborating institutions, academic achievements, and license.
2. Replace the original AVA architecture image with a chart example pic : 
  <div align="center">AVA intro
  </div>
3. According to the suggestions, the markdown syntax errors were corrected, the case was optimized, the 'development' section was deleted, and the description and usage of the AVA components were optimized.

### CODE REVIEW

File: README.zh-CN.md [hunk 1]
```diff
@@ -12,38 +12,103 @@
 </div>
 
 ----
-
-<a href="https://ava.antv.antgroup.com"><img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*rXVYRJ0EMDsAAAAAAAAAAAAADmJ7AQ/original" align="left"  width="180"  hspace="10" vspace="6"></a>
-
-**AVA** (<img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*QzIsSrfsCW0AAAAAAAAAAAAADmJ7AQ/original" width="16"> Visual Analytics) 是为了更简便的可视分析而生的技术框架。 其名称中的第一个 **A** 具有多重涵义：一个智能驱动（*AI driven*）、自动化（*Automated*）、支持增强分析（*Augmented*）的可视分析解决方案。
-
+## ❓ 什么是AVA
+[AVA](https://github.com/antvis/AVA)（<img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*QzIsSrfsCW0AAAAAAAAAAAAADmJ7AQ/original" width="16"> Visual Analytics）是为了更简便的可视分析而生的技术框架。第一个 **A** 具有多重涵义：智能驱动（*AI driven*）、自动化（*Automated*）、支持增强分析（*Augmented*）, **VA** 代表可视分析。它可以辅助用户进行**数据处理**、提取**数据洞察**、实现**图表的推荐和自动生成**，此外它还可以**优化已有的图表**。
```
[the remark this answers is not in the document]
Comment (neoddish) [reply README.zh-CN.md:16]: @chenying2929 maybe we could add the alt

File: README.zh-CN.md [hunk 2]
```diff
@@ -12,38 +12,103 @@
 </div>
 
 ----
-
-<a href="https://ava.antv.antgroup.com"><img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*rXVYRJ0EMDsAAAAAAAAAAAAADmJ7AQ/original" align="left"  width="180"  hspace="10" vspace="6"></a>
-
-**AVA** (<img src="https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*QzIsSrfsCW0AAAAAAAAAAAAADmJ7AQ/original" width="16"> Visual Analytics) 是为了更简便的可视分析而生的技术框架。 其名称中的第一个 **A** 具有多重涵义：一个智能驱动（*AI driven*）、自动化（*Automated*）、支持增强分析（*Augmented*）的可视分析解决方案。
-
+## ❓ 什么是AVA
```
Comment (neoddish) [inline README.zh-CN.md:15]: What is your consideration about adding emojis before the title?

Comment (neoddish) [inline README.zh-CN.md:15]:
```suggestion
## ❓ 什么是 AVA
```

File: README.zh-CN.md [hunk 3]
```diff
@@ -12,38 +12,103 @@
...
 </div>
 
-## npm 包
-
-### [@antv/ava](https://github.com/antvis/AVA/blob/master/packages/ava)
-
-### [@antv/ava-react](https://github.com/antvis/AVA/tree/master/packages/ava-react)
-
-## 参与贡献 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+## 📁 AVA的组成
```
Comment (neoddish) [inline README.zh-CN.md:23]:
```suggestion
## 📁 AVA 的组成
```

File: README.zh-CN.md [hunk 4]
```diff
@@ -12,38 +12,103 @@
...
+AVA包含2个包，分别为 [@antv/ava](https://www.npmjs.com/package/@antv/ava) 和 [@antv/ava-react](https://www.npmjs.com/package/@antv/ava-react)：
```
Comment (neoddish) [inline README.zh-CN.md:24]:
```suggestion
AVA 包含2个包，分别为 [@antv/ava](https://www.npmjs.com/package/@antv/ava) 和 [@antv/ava-react](https://www.npmjs.com/package/@antv/ava-react)：
```

File: README.zh-CN.md [hunk 5]
```diff
@@ -12,38 +12,103 @@
...
++ [@antv/ava](https://www.npmjs.com/package/@antv/ava) ：AVA核心包，包含四大主要模块：
```
Comment (neoddish) [inline README.zh-CN.md:25]:
```suggestion
* [@antv/ava](https://www.npmjs.com/package/@antv/ava) ：AVA 核心包，包含四大主要模块：
```

check the `.markdownlint.json` of the repo, we only allow asterisk for markdown ul-style.

btw, you may want to install the markdownlint extension in vscode to show you the linting.

File: README.zh-CN.md [hunk 6]
```diff
@@ -12,38 +12,103 @@
...
+  + **data (数据处理)** ：数据处理器。用于数据集统计分析和处理。
+  + **insight (智能洞察)**：自动洞察器。自动地从多维数据中发现数据洞察。
+  + **ckb (图表知识库)**：图表知识库。基于经验总结的关于可视化和图表的各种基本知识和观察，它是智能图表推荐的基石。
+  + **advisor (图表推荐和自动生成)**：图表建议官。基于数据和分析需求来推荐图表类型和具体的图表细节设置，也可对既有的图表进行图表优化。
```
Comment (neoddish) [inline README.zh-CN.md:29]:
```suggestion
  + **data (数据处理)** ：数据处理模块。用于数据集统计分析和处理。
  + **insight (智能洞察)**：自动洞察模块。自动地从多维数据中发现数据洞察。
  + **ckb (图表知识库)**：图表知识库模块。基于经验总结的关于可视化和图表的各种基本知识和观察，它是智能图表推荐的基石。
  + **advisor (图表推荐和自动生成)**：图表推荐模块。基于数据和分析需求来推荐图表类型和具体的图表细节设置，也可对既有的图表进行图表优化。
```

File: README.zh-CN.md [hunk 7]
```diff
@@ -12,38 +12,103 @@
...
++ <a href='https://ava.antv.antgroup.com/api/ckb/ckb' target='_blank'>API文档</a>
++ <a href='https://ava.antv.antgroup.com/examples' target='_blank'>图表示例</a>
+
+## 💻 开始使用
+可以通过 NPM 包管理器来安装。
+```bash
+$ npm install @antv/ava
+$ npm install @antv/ava-react
+```
+
```
Comment (neoddish) [inline README.zh-CN.md:48]:
```suggestion

## 💻 开始使用

可以通过 NPM 包管理器来安装。

```bash
$ npm install @antv/ava
$ npm install @antv/ava-react
```

```

Comment (neoddish) [reply README.zh-CN.md:48]: check markdownlint

File: README.zh-CN.md [hunk 8]
```diff
@@ -57,3 +122,13 @@ Chen, Q., Sun, F., Xu, X., Chen, Z., Wang, J. and Cao, N., 2021. VizLinter: A Li
...
+## ❗ 许可证
+
+MIT@[AntV](https://github.com/antvis).
+
+<!-- ## npm 包
+
+### [@antv/ava](https://github.com/antvis/AVA/blob/master/packages/ava)
+
+### [@antv/ava-react](https://github.com/antvis/AVA/tree/master/packages/ava-react) -->
```
Comment (neoddish) [inline README.zh-CN.md:134]: why the comments?

File: README.zh-CN.md [hunk 9]
```diff
@@ -12,38 +12,103 @@
...
+
+# build the packages
+$ npm run build
+
+# run playground
+$ npm run setup:playground
+
+# run site
+$ npm run setup:site
+```
```
Comment (neoddish) [inline README.zh-CN.md:93]: This part could be in `./CONTRIBUTING.zh-CN.md`

and these commands may not be Required

File: README.zh-CN.md [hunk 10]
```diff
@@ -12,38 +12,103 @@
...
+
+const firstInsight = getInsights(data).insights[0]
+
+ReactDOM.render(
+  <>
+    <InsightCard insightInfo={firstInsight} visualizationOptions={{ lang: 'zh-CN' }} />
+  </>,
+  mountNode,
+);
+```
```
Comment (neoddish) [inline README.zh-CN.md:78]: Personally, I would prefer each of these two packages to have their own example.

**Review by neoddish — CHANGES_REQUESTED** [review]

**Comment by neoddish** [comment]
> 2. add a image

could u plz preview the img in the PR comment? thx!

**Comment by neoddish** [comment]
btw, plz change the msg of these 2 commits:

* remove `wip:`, this should not be in a commit msg title
* `docs(readme):` would be fine

File: README.zh-CN.md (hunk 2 above)
Comment (chenying2929) [reply README.zh-CN.md:15]: I referenced G2. https://github.com/antvis/G2/blob/v5/README.md

Comment (chenying2929) [reply README.zh-CN.md]: done～

File: README.zh-CN.md (hunk 3 above)
Comment (chenying2929) [reply README.zh-CN.md]: done～

File: README.zh-CN.md (hunk 4 above)
Comment (chenying2929) [reply README.zh-CN.md]: done～

File: README.zh-CN.md (hunk 5 above)
Comment (chenying2929) [reply README.zh-CN.md]: done～

File: README.zh-CN.md (hunk 6 above)
Comment (chenying2929) [reply README.zh-CN.md]: done～

File: README.zh-CN.md (hunk 7 above)
Comment (chenying2929) [reply README.zh-CN.md]: done～

File: README.zh-CN.md (hunk 8 above)
Comment (chenying2929) [reply README.zh-CN.md]: I removed it～

File: README.zh-CN.md (hunk 9 above)
Comment (chenying2929) [reply README.zh-CN.md]: ok～i removed it

File: README.zh-CN.md (hunk 10 above)
Comment (chenying2929) [reply README.zh-CN.md]: done～

**Comment by chenying2929** [comment]
> > 2. add a image
> 
> could u plz preview the img in the PR comment? thx!

done～

File: README.zh-CN.md (hunk 2 above)
Comment (neoddish) [reply README.zh-CN.md]: but why

Comment (chenying2929) [reply README.zh-CN.md]: From the perspective of rigor, the emoji should be removed. done～

**Review by neoddish — APPROVED** [review]
LGTM

**Pull request merged**